python_modules/dagster/dagster/_core/pipes/utils.py

```diff
@@ -618,7 +650,21 @@ def _join_thread(thread: Thread, thread_name: str) -> None:
         raise DagsterPipesExecutionError(f"Timed out waiting for {thread_name} thread to finish.")
 
 
-def extract_message_or_forward_to_file(handler: "PipesMessageHandler", log_line: str, file: TextIO):
+def forward_only_logs_to_file(log_line: str, file: IO[str]):
```

Comment:
Seems like this isnt actually called anywhere? Why is it here?

Reply:
Sorry, it was a leftover from a PR split. Removed.